.github/workflows/manual-build.yaml

```diff
@@ -11,7 +12,8 @@ on:
         default: ''
 env:
   GO_VERSION: "1.18"
-  IMG: "quay.io/lvmo/lvm-operator"
+  OPERATOR_IMG: "quay.io/lvmo/lvm-operator"
```

Comment:
go version can be removed and replaced with a go.mod file. https://github.com/red-hat-storage/odf-operator/blob/main/.github/workflows/docker-push.yaml#L23 for more information.

Reply:
Done